src/config.js

```diff
@@ -36,6 +36,23 @@ const editorialFrontendDomain = () => {
   }
 };
 
+const gaTrackingId = () => {
+  if (process.env.NODE_ENV !== 'production') {
+    return '';
+  }
+
+  switch (ndlaEnvironment) {
+    case 'local':
+      return '';
+    case 'dev':
+      return '';
+    case 'prod':
+      return 'UA-9036010-36';
+    default:
+      return 'UA-9036010-36';
```

Comment:
Er det ikkje bedre å ha default til tom?

Reply:
Tja, jo, i ndla-frontend/learningpath-frontend har vi en egen key for test/staging. Kan kanskje la den stå tom inntil vi eventuelt har det?